talk/expert/modules.tex

```diff
@@ -401,7 +401,7 @@
       \begin{itemize}
         \item I.e.\ the header does not require setup of preprocessor state before inclusion
       \end{itemize}
-      \item All \cpp20 headers are importable
+      \item All standard library headers are importable
```

Comment:
Oh wait, this should still read `All \cpp20 standard library headers`. Because (as in the next sentence) the C standard library headers are not importable.

Reply:
I will add this to a subsequent PR.

Reply:
you're right, didn't think about the C headers!